tests/fsharp/core/printing/output.1000.stdout.bsl

```diff
@@ -5,44 +5,44 @@
 
 > val repeatId: string = "B"
 
-namespace FSI_0005
-  val x1: int
-  val x2: string
-  val x3: 'a option
-  val x4: int option
-  val x5: 'a list
-  val x6: int list
-  val x7: System.Windows.Forms.Form
-  val x8: int[,]
-  val x9: Lazy<string>
-  type ClassInFile1 =
-    new: unit -> ClassInFile1
-
-namespace FSI_0006
-  val x1: int
-  val x2: string
-  val x3: 'a option
-  val x4: int option
-  val x5: 'a list
-  val x6: int list
-  val x7: System.Windows.Forms.Form
-  val x8: int[,]
-  val x9: Lazy<string>
-  type ClassInFile1 =
-    new: unit -> ClassInFile1
-
-namespace FSI_0006
-  val x1: int
-  val x2: string
-  val x3: 'a option
-  val x4: int option
-  val x5: 'a list
-  val x6: int list
-  val x7: System.Windows.Forms.Form
-  val x8: int[,]
-  val x9: Lazy<string>
-  type ClassInFile2 =
-    new: unit -> ClassInFile2
+module FSI_0005.TestLoadFile
```

Comment:
Interesting that these are now coking bout like this, I hadn't quite expected that but it makes sense